kolibri/content/management/commands/populate_contentdb.py

```diff
@@ -0,0 +1,83 @@
+        models.MimeType.objects.using(db_name).create(readable_name='exercise_highres', machine_name='Ed-209')
+
+        # populate License
+        WTFPL = models.License.objects.using(db_name).create(license_name='WTFPL')
+        GNU = models.License.objects.using(db_name).create(license_name='GNU')
+        CC = models.License.objects.using(db_name).create(license_name='CC')
+        models.License.objects.using(db_name).create(license_name='MIT')
+
+        # populate ContentMetadata
+        root = models.ContentMetadata.objects.using(db_name).create(
+            title='root', parent=None, license=WTFPL, kind='topic',
+            description='balbla1', slug='slut_1', total_file_size=21, available=False, content_id=uuid.uuid4())
+        c1 = models.ContentMetadata.objects.using(db_name).create(
+            title='c1', parent=root, license=WTFPL, kind='video',
+            description='balbla2', slug='slut_2', total_file_size=22, available=False, content_id=uuid.uuid4())
+        c2 = models.ContentMetadata.objects.using(db_name).create(
+            title='c2', parent=root, license=GNU, kind='topic',
+            description='balbla3', slug='slut_3', total_file_size=23, available=False, content_id=uuid.uuid4())
+        c2c1 = models.ContentMetadata.objects.using(db_name).create(
+            title='c2c1', parent=c2, license=CC, kind='exercise',
+            description='balbla4', slug='slut_4', total_file_size=23, available=False, content_id=uuid.uuid4())
+        models.ContentMetadata.objects.using(db_name).create(
+            title='c2c2', parent=c2, license=GNU, kind='topic',
+            description='balbla5', slug='slut_5', total_file_size=23, available=False, content_id=uuid.uuid4())
+        models.ContentMetadata.objects.using(db_name).create(
+            title='c2c3', parent=c2, license=GNU, kind='topic',
+            description='balbla5', slug='slut_6', total_file_size=24, available=False, content_id=uuid.uuid4())
+
+        # populate Format
+        format_1 = models.Format.objects.using(db_name).create(contentmetadata=c1, mimetype=video_H, available=False, format_size=102, quality="high")
+        format_2 = models.Format.objects.using(db_name).create(contentmetadata=c1, mimetype=video_L, available=False, format_size=51, quality="low")
+        format_3 = models.Format.objects.using(db_name).create(contentmetadata=c2c1, mimetype=ex_L, available=False, format_size=46, quality="high")
+
+        # populate File
+        models.File.objects.using(db_name).create(format=format_1)
+        models.File.objects.using(db_name).create(format=format_2)
+        models.File.objects.using(db_name).create(format=format_3)
+        models.File.objects.using(db_name).create(format=format_3)
+
+        # populate Content Relationship
+        models.PrerequisiteContentRelationship.objects.using(db_name).create(relationship_type='prerequisite', contentmetadata_1=root, contentmetadata_2=c1)
```

Comment:
If we have separate objects for the relationships, we no longer need a relationship type field.